### ISSUE TITLE
使用Glide插件，加载apng动图没效果

### ISSUE THREAD

**Issue by chauyon** [penfeizhou/APNG4Android#226]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | e.g.  1.0.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by GHpeter** [comment]
I also encountered the same problem when using the Apng library. I tried to add GlideModule annotation of Glide, but still could not animate the Apng format images. I thought it was because the version of the dependency was too low (2.11.0). After trying to upgrade to the latest version 3.0.1, the problem still could not be solved. Have you solved the problem? If you have solved it, please share the code so that I can see the solution. Thank you.

**Comment by penfeizhou** [comment]
Please check if you add anpg as drawable resource. Apt would modify apng file in that case.

**Comment by GHpeter** [comment]
资源测试过是没有问题的，在web端，iOS端都有apng的动画效果，因为项目原因，现集成依赖如下：
 implementation ("com.github.bumptech.glide:glide:4.16.0")
annotationProcessor ("com.github.bumptech.glide:compiler:4.16.0")
implementation ("com.github.penfeizhou.android.animation:glide-plugin:2.11.0")
implementation ("com.github.penfeizhou.android.animation:apng:2.11.0")

已新增GlideModule

import com.bumptech.glide.annotation.GlideModule
import com.bumptech.glide.module.AppGlideModule

@GlideModule
class MyGlideModule:AppGlideModule()


调用代码如下：
val aPngUrl ="xxxxx.png"  //图片涉及项目信息不便展示
Glide.with(this).load(aPngUrl).into(binding.imgApng)

**Comment by penfeizhou** [comment]
如果有问题，请用可以复现的demo工程来说明

**Comment by GHpeter** [comment]
上述代码即为demo中的代码，可将apngurl内容替换为：val aPngUrl ="https://img0.baidu.com/it/u=3805650362,3133230436&fm=253&fmt=auto&app=138&f=PNG?w=166&h=120"

在Apng4Android库中的demo代码中未发现有通过使用Glide加载网络图片实现动效的效果，能否补充看下效果

**Comment by penfeizhou** [comment]
Demo中有用glide加载网络图片的例子，只是链接目前是本地链接，替换成网络链接即可加载。

**Comment by penfeizhou** [comment]
<img width="1164" alt="image" src="https://github.com/user-attachments/assets/e9c579f2-4c74-44b6-b3d3-55fe3ab73a2b">
<img width="262" alt="image" src="https://github.com/user-attachments/assets/ecdd6746-7ef9-4e58-a4c7-e0d7e451d959">

**Comment by GHpeter** [comment]
好的，我再检查下，按理说只要添加了@GlideModule这个注解的类，编译成功就可以了对吧

**Comment by penfeizhou** [comment]
你用的kotlin应该用 `kapt`而不是`annotationProcessor`吧

**Comment by GHpeter** [comment]
嗯，这个我再调整看下效果，多谢大佬指点

**Comment by GHpeter** [comment]
感谢大佬指点，错误原因确是因为你说的这个：
用的kotlin应该用 kapt而不是annotationProcessor

目前已能正常显示效果，感谢！
